website/src/pages/en/about.mdx

```diff
@@ -1,67 +1,55 @@
 ---
 title: About The Graph
+description: This page summarizes the core concepts and basics of The Graph Network.
 ---
 
-## What is The Graph?
+## Explanation
 
-The Graph is a powerful decentralized protocol that enables seamless querying and indexing of blockchain data. It simplifies the complex process of querying blockchain data, making dapp development faster and easier.
+### What is The Graph?
 
-## Understanding the Basics
+The Graph is a decentralized protocol for indexing and querying blockchain data. Its suite includes [Subgraphs](/en/subgraphs/developing/subgraphs/), [Substreams](/en/substreams/introduction/), [Token API BETA](/en/token-api/quick-start/), and tools like [Graph Explorer](/en/subgraphs/explorer/) and [Subgraph Studio](/en/subgraphs/developing/deploying/using-subgraph-studio/).
 
-Projects with complex smart contracts such as [Uniswap](https://uniswap.org/) and NFTs initiatives like [Bored Ape Yacht Club](https://boredapeyachtclub.com/) store data on the Ethereum blockchain, making it very difficult to read anything other than basic data directly from the blockchain.
+The Graph supports [90+ blockchains](/en/supported-networks/), enhancing dapp development and data retrieval.
```

Comment:
```suggestion
The Graph supports [90+ blockchains](/supported-networks/), enhancing dapp development and data retrieval.
```
(also, is it weird to say "blockchains" here when the page we link to calls them "networks"?)